convert2rhel/unit_tests/example_test.py

```diff
@@ -19,29 +19,28 @@
 This is an example test file containing a simple test.
 """
 
-# Required imports:
-import unittest
-
 from convert2rhel import unit_tests  # Imports unit_tests/__init__.py
 from convert2rhel import utils
 
 
-class TestExample(unittest.TestCase):
-    class RunSubprocessMocked(unit_tests.MockFunction):
-        def __call__(self, *args, **kwargs):
-            """
-            Implementation of the simplest behavior of a mocked function -
-            ignore the input parameters and just return fake value.
-            """
-            self.prefix = "this ain't "
-            self.ret = (self.prefix + self.ret[0], self.ret[1])
-            return self.ret
-
-    @unit_tests.mock(utils, "run_subprocess", RunSubprocessMocked())
-    def test_example(self):
-        # Set a tuple to be returned by the RunSubprocessMocked function
-        utils.run_subprocess.ret = ("ls output", 0)
-        output, ret_code = utils.run_subprocess("ls -l")
-        self.assertEqual(output, "this ain't ls output")
-        self.assertEqual(ret_code, 0)
-        self.assertEqual(utils.run_subprocess.prefix, "this ain't ")
+class RunSubprocessMocked(unit_tests.MockFunction):
+    def __call__(self, *args, **kwargs):
+        """
+        Implementation of the simplest behavior of a mocked function -
+        ignore the input parameters and just return fake value.
+        """
+        self.prefix = "this ain't "
+        self.ret = (self.prefix + self.ret[0], self.ret[1])
+        return self.ret
```

Comment:
-- Not something to fix, just a note --

Just a quick note: We have an Issue in our Jira board to transform classes/functions likes this into fixtures in the conftest.py file. 

Maybe it could be a good idea for us, in the future, to move this class to be a fixture and add a couple of examples here on how to use it.